Client/src/Pages/Monitors/index.jsx

```diff
@@ -152,17 +154,52 @@ const Monitors = () => {
 
   const down = monitorState.monitors.length - up;
 
+  const [sortOrder, setSortOrder] = useState(-1);
+  const [sortedMonitors, setSortedMonitors] = useState(monitorState.monitors);
+  const [isSorted, setIsSorted] = useState(false);
+
+  useEffect(() => {
+    const savedSortOrder = localStorage.getItem('monitorSortOrder');
+    const parsedSortOrder = parseInt(savedSortOrder, 10) || 1;
+  
+    if (savedSortOrder) {
+      setSortOrder(parsedSortOrder);
+      const monitors = [...monitorState.monitors].sort(a => {
+        return a.status ? -1 * parsedSortOrder : 1 * parsedSortOrder;
+      });
+      setSortedMonitors(monitors);
+      setIsSorted(true);
+    } else {
+      setSortedMonitors(monitorState.monitors);
+    }
+  }, [monitorState.monitors]);
+  
+  
+  const handleSort = () => {
+    const newSortOrder = sortOrder * -1;
+    setSortOrder(newSortOrder);
+  
+    const monitors = [...monitorState.monitors].sort(a => {
+      return a.status ? -1 * newSortOrder : 1 * newSortOrder;
+    });
```

Comment:
I think this can be simplified somewhat to

```
    setSortOrder(-sortOrder);

    const monitors = [...monitorState.monitors].sort((a) => {
        return a.status ? -sortOrder : sortOrder;
    });
    ```
I don't think the `newSortOrder` var is necessary here.  If I've missed something let me know :thinking: 
    
We can save a bit of verbosity by directly inverting the sort order with the `-` operator too instead of multiplying by `-1`

May as well simplify wherever we can :joy:

Reply:
Thanks for this, you taught me something! I went ahead and used this logic and it works flawlessly. Went ahead and removed the localStorage as well.